model/preference.go

```diff
@@ -116,7 +118,7 @@ func (o *Preference) PreUpdate() {
 
 		// blank out any invalid theme values
 		for name, value := range props {
-			if name == "image" || name == "type" || name == "codeTheme" {
+			if name == "image" || name == "type" || name == "codeTheme" || name == "colorScheme" {
```

Comment:
@streamer45 possible values of `colorScheme` are `light` / `dark`. It is used by client apps to categorize themes in theme selection form:
![Screen Shot 2021-09-02 at 17 55 32](https://user-images.githubusercontent.com/8155563/131878140-986cc66e-ba43-44bb-a579-eca34d62838e.png)

Reply:
Thanks :+1:  I am wondering if the name is appropriate. I identify a color scheme as a set of colors and you can have many of those. Maybe `lightMode` would be more fitting?

Reply:
@streamer45 React native uses `colorScheme` https://reactnative.dev/docs/appearance#getcolorscheme. I named it same way to keep "ubiquitous language".

Reply:
For me, it is no problem to change it 😉

Reply:
@streamer45 I think `isLight` would be better (this is theme's property so in the webapp we have `theme.isLight`)

Reply:
> React native uses colorScheme https://reactnative.dev/docs/appearance#getcolorscheme. I named it same way to keep "ubiquitous language".

[TIL](https://dictionary.cambridge.org/dictionary/english/til) :)

> I think isLight would be better (this is theme's property so in the webapp we have theme.isLight)

Yeah that's good as well but I have no strong opinion.  We can also keep it as is since now I know why it was named that way and I am all for consistency :) @esethna , thoughts?